### ISSUE TITLE
About baseline, help

### ISSUE THREAD

**Issue by chenpp1999** [LMZZML/ST-AE#1]
Hello, I have read your paper and I think it is a great paper. I am currently studying this direction as well and I have a question that I hope you can help me with. About the baseline, do I have to do all the baseline experiments by myself. Is that too much work and is there an easy way to do it? Thank you for your help!

**Comment by LMZZML** [comment]
If you feel there are too many baseline experiments, it is possible to directly reference the baseline experimental results from other papers, given that the experimental settings are the same as those in the existing research. However, I highly recommend running the baseline methods yourself. It provides a better understanding of the baseline methods, and it allows you to verify the accuracy of results reported in published papers. It may not be a very difficult task, and if you plan to work in this field for a long time, it is essential to maintain these results. I hope this can be helpful to you.

**Comment by chenpp1999** [comment]
Thank you very much! So if I do my own experiments with baseline, then do I just need to replace my model file with baseline, or do I have to use baseline its own trianer and other files.

**Comment by yiiixuan12** [comment]
Hi, I read your article and now I want to reproduce your code.
 I currently ran gernerate_training_data.py first, and there are three npz files for training set, test set and validation set under data/PEMS08 folder. Then I tried to run the python ae_train.py command, which prompted the runtime error：Expected 2D (unbatched) or 3D (batched) input to conv1d, but got input of size: [64, 32, 170, 13], appears in: filter = self.filter_convs[i](residual) code line
